File: python/get-senders.py
import requests
import json

url = "<REPLACE_WITH_PROD_URL>/v1/senders"

params = {
    "query": "searchTerm",        
    "status": "PENDING",         
    "allAccounts": "true"       
}

headers = {
    'Authorization': 'Bearer <REPLACE_WITH_YOUR_API_KEY>',
    'Content-Type': 'application/json'
}

response = requests.get(url, params=params, headers=headers)
data = response.json()
print(json.dumps(data))

File: js/get-senders.js
const prodUrl = "<REPLACE_WITH_PROD_URL>"
const token = "<YOUR_TOKEN>"

const getSenders = async () => {
  try {
    const params = new URLSearchParams({
      query: "searchTerm",
      status: "PENDING",
      allAccounts: "true",
    })

    const url = `${prodUrl}/v1/senders?${params}`

    const response = await fetch(url, {
      method: "GET",
      headers: {
        Authorization: `Bearer ${token}`,
        "Content-Type": "application/json",
      },
    })

    const responseData = await response.json()
    console.log(JSON.stringify(responseData))
  } catch (error) {
    console.log(error)
  }
}

getSenders()
